client/x/evmstaking/keeper/abci.go

```diff
@@ -110,6 +111,19 @@ func (k *Keeper) EndBlock(ctx context.Context) (abci.ValidatorUpdates, error) {
 			return nil, errors.Wrap(err, "burn coins")
 		}
 
+		log.Debug(ctx, "Adding undelegation to withdrawal queue",
+			"delegator", entry.delegatorAddress,
+			"validator", entry.validatorAddress,
+			"max_amount", maxAmount.String())
+		if entry.amount.LT(maxAmount) {
+			log.Warn(ctx, "Undelegation amount is less than max amount",
```

Comment:
Seems the check can never return True and the log would never be printed out, because the `maxAmount` was set to the value of `entry.amount` at Ln 99 - 101, hence the `entry.amount` can never less than `maxAmount`. sounds right?

It appears that the check will never return True, and the log will never be printed because `maxAmount` is assigned the value of `entry.amount` on lines 99–101. As a result, `entry.amount` can never be less than `maxAmount`. Does that sound correct?

Reply:
Thanks, fixed in dfdd687